keyboards/idobao/id75/keymaps/paryz/keymap.c

```diff
@@ -0,0 +1,149 @@
+/* ht 2018 Milton Griffin
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+#include QMK_KEYBOARD_H
+
+#ifdef RGB_MATRIX_ENABLE
+led_config_t custom_led_config = { {
```

Comment:
You know, you could just move all of this to the keyboard_post_init_user function, as: 
```c
    g_led_config = { {
```

Reply:
Well, yes I guess I could. I believe that's what the compiler will do anyways, to optimize it.

Reply:
Extremely unlikely -- it's only going to have side effects after `keyboard_post_init_user()`. Prior to that, the keyboard-supplied values will be present.

I'll wait to see if you want to change things before merge -- your call.

Reply:
Yup, thank you for waiting, changed it. None of the changes made are breaking, so I think we can go ahead :)